index.html

```diff
@@ -0,0 +1,218 @@
+          <h6 class="homepage-location">@ Free Virtual program</h6>
+        </div>
+      </section>
+      <section class="homepage-programs" id="programs">
+        <div class="homepage-programs-container">
+          <h2>Main Program</h2>
+          <hr />
+          <div class="homepage-programs-content">
+            <article class="programs-content">
+              <img src="./assets/icons/lecture.png" alt="Lecture" />
+              <h4>Lecture</h4>
+              <p>
+                This free live summit event will welcome hundreds of digital
+                delegates from across Ghana and around the world to discuss the
+                future of employment, entrepreneurship, and education in Ghana
+                as the country faces its highest level of youth unemployment.
+              </p>
+            </article>
+            <article class="programs-content">
+              <img src="./assets/icons/technology.png" alt="Exhibition" />
+              <h4>Exhibition</h4>
+              <p>
+                Participants will have an opportunity to showcase their digital
+                products and inventions to entrepreneurs, investors, digital
+                marketers and creatives.
+              </p>
+            </article>
+            <article class="programs-content">
+              <img src="./assets/icons/forum.png" alt="Forum" />
+              <h4>Forum</h4>
+              <p>
+                Participants will have an opportunity to discuss the future of
+                technology in Africa with entrepreneurs, investors, digital
+                marketers and creatives.
+              </p>
+            </article>
+            <article class="programs-content special3">
+              <img src="./assets/icons/workshop.png" alt="Workshop" />
+              <h4>Workshop</h4>
+              <p>
+                Insights and initiatives leveraging tech and innovative
+                inclusive ways to accelerate the future of Ghana will be
+                discussed.
+              </p>
+            </article>
+            <article class="programs-content">
+              <img src="./assets/icons/networking.png" alt="Networking" />
+              <h4>Networking</h4>
+              <p>
+                The 2022 edition is seeking to welcome over 10000 global digital
+                delegates. There are opportunities to network with tech industry
+                gurus accross the world. Networking is the masterstroke when it
+                comes to marketing your business, and nothing can beat
+                connecting with your audience personally.
+              </p>
+            </article>
+          </div>
+          <button type="button" id="join">
+            Register Ghana Tech Summit 2022 for free
+          </button>
+        </div>
+      </section>
+      <section class="homepage-speakers" id="speakers">
+        <div class="speakers-container">
+          <h2>Featured Speakers</h2>
```

Comment:
- [ ] In this section, if you check the original Figma template, you will realize that there is a little grid on each image of the featured speakers, so please, in order to stick to the given design as much as possible, implement this feature as well.

Reply:
- [ ] Kindly work on this change as requested by the previous reviewer. From the design given, there is a grid floated design at the top left of each speakers' image. Kindly fix that.

**In the design below I marked the sections in green circles so you can spot it.
![image](https://user-images.githubusercontent.com/50941074/164514534-cea1e255-f45f-4a05-91f4-0a60f3ed2041.png)

**Your Design**
![image](https://user-images.githubusercontent.com/50941074/164514704-2cfdf465-c26c-4829-98bb-181f9a9d30a7.png)